src/components/container/ConnectedDevices/ConnectedDevices.tsx

```diff
@@ -3,13 +3,8 @@ import MyDataHelps, { DeviceDataNamespace, DeviceDataPoint, DeviceDataPointQuery
 import "./ConnectedDevices.css"
 import language from "../../../helpers/language"
 import { CardTitle } from '../../presentational';
-import { faBatteryEmpty } from "@fortawesome/free-solid-svg-icons/faBatteryEmpty"
-import { faBatteryFull } from "@fortawesome/free-solid-svg-icons/faBatteryFull"
-import { faBatteryHalf } from "@fortawesome/free-solid-svg-icons/faBatteryHalf"
-import { faBatteryQuarter } from "@fortawesome/free-solid-svg-icons/faBatteryQuarter"
-import { faWeightScale } from "@fortawesome/free-solid-svg-icons/faWeightScale"
-import formatRelative from 'date-fns/formatRelative'
-import parseISO from 'date-fns/parseISO'
+import { faBatteryEmpty, faBatteryFull, faBatteryHalf, faBatteryQuarter, faWeightScale } from "@fortawesome/free-solid-svg-icons/"
```

Comment:
I think you need to drop the `/` at the end of this import or the storybook fails to load.

Reply:
Nice find. Removed.